tests/pricefeed_test.py

```diff
@@ -1,81 +1,74 @@
 # pricefeed_test.py
 from datetime import datetime, timedelta
 
-import tests
-from nibiru import common
+from tests import dict_keys_must_match, transaction_must_succeed
 
-from . import CONFIG
 
+class TestPricefeed:
```

Comment:
Same, we could remove this class

Reply:
Indeed, this is just an extra indent now.